src/hockeypuck/hkp/storage/storage.go

```diff
@@ -83,7 +83,7 @@ type Inserter interface {
 
 	// Insert inserts new public keys if they are not already stored. If they
 	// are, then nothing is changed.
-	Insert([]*openpgp.PrimaryKey) (int, error)
+	Insert([]*openpgp.PrimaryKey) (int, int, error)
```

Comment:
Done.

Reply:
I changed the documentation of `<u>` again, after the new commit. To avoid multiple commits, the current form is:
```
	// <u>   is the number of keys updated, if any. When a PrimaryKey in the input is
	//       already in the DB (same rfingerprint), but has a different md5 (e.g., because
	//       of a non-overlapping set of signatures), the keys are merged together. If
	//       signatures, attributes etc are a subset of those of the key in the DB, the
	//       input key is considered a duplicate and there is no update.
```